students/README.md

```diff
@@ -0,0 +1,86 @@
+- Promote diversity and inclusion in the way that universities and students in other countries can contribute.
+ ## To be done
+- List CNCF projects that could be interesting in this student toolkit
+- Define the minimum skills to develop in potential students contributors
+- Define the steps to get involved students into participate into GSoC related to CNCF projects
+- List professors, universities that can be interested in this program.
+- Generate a vault of projects, features of issues that the students/students can be involved in
+- If possible create a CNCF conference for students to promote students participation
+- Gather students opinions about this kind of opportunities to enrich this student kit
+- Work together with universities to update their syllabus in the way to include cloud native theory in their courses.
+- Create Bi-Weekly Work Meetings to develop and promote this toolkit
+ 
+## Students Event Examples
+### Students Presenting a Class Project
+The students of USAC, University of Guatemala presented a project about a high concurrency system that displays real time information about Covid-19, this project used Cloud Native technologies such as Kubernetes, Linkerd, gRPC, Prometheus, and others. Charles Pretzer from Linkerd Community and Field Engineer of Buoyant, helped us answering questions and promoted students to Join the Linkerd Community. This was one of many sessions that try to involve students into the CNCF ecosystems, we also promoted the hashtag #cncfstudents to share students work on the social networks, Sergio's role was to organize the event and help the students to present the project to the linkerd people.
+- [Realtime COVID-19 Display System Using Linkerd](https://www.youtube.com/watch?v=XWlpS78wRks)
+ 
+### Open Syllabus and project
+This repository is an initiative to develop an open content and resources for students to learn about cloud native technologies.
+- [Operating System Syllabus and Projects](https://github.com/sergioarmgpl/operating-systems-usac-course)
+ 
+### CNCF Students Day
+This event was hosted in Nicaragua, as an online event, to promote CNCF technologies. The students from Guatemala and some other speakers that are organizers of CNCF Community Groups participated giving workshops, and talks focused on containers, Kubernetes, and cloud native in general. Most of the attendees were students.
+
+- [Cloud Native Students Day URACCAN](https://community.cncf.io/events/details/cncf-cloud-nativegt-presents-cloud-native-students-day-uraccan/)
+ 
+## How to promote this toolkit
+This toolkit or students movement should be promoted using the hashtag:
+- #CNCFStudents  
+  
+Please share all students events with this hashtag to grow as a CNCF Students movement
+ 
+## Contact
+You can contact us in the CNCF #students slack channel for more information.
+- [CNCF Slack](https://slack.cncf.io/)
+ ### Team
+- Sergio Méndez, USAC
```

Comment:
I think someone from CNCF staff should be responsible here as well. @xmulligan, you?